framework/components/ema-content-slider/ema-content-slider.twig

```diff
@@ -0,0 +1,29 @@
+{#
+  Accepts:
+    -
+#}
+
+{# Internal properties #}
+{% set _css_class = 'ema-content-slider' %}
```

Comment:
We should allow extra attributes too

Reply:
Line 22

Reply:
You add extra_attributes for list items, but not for the content slider itself